src/Util/Instantiator.php

```diff
@@ -27,7 +29,7 @@ public static function instantiate(string $class): object
     {
         $r = new \ReflectionClass($class);
         if ($r->isAbstract()) {
-            throw new \RuntimeException(sprintf('Cannot initialize abstract class: %s', $class));
+            throw new AbstractClassException(sprintf('Cannot initialize abstract class: %s', $class));
```

Comment:
```suggestion
            throw new AbstractClassException($class);
```

Then you can update `AbstractClassException`'s constructor which will create the error message.

Reply:
This could be great indeed

Reply:
Done, sounds more logical.